packages/network-graph/components/network-graph/src/network-graph.ts

```diff
@@ -0,0 +1,177 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+import ForceGraph2D, { GraphData } from 'force-graph';
+import { CustomLinks, CustomNode } from '../defs';
+
+/**
+ * Graph component to render graph based on nodes and links
+ */
+export class NetworkGraph extends LitElement {
+  @property({ attribute: 'label', type: String })
+  label = '';
+
+  @property({ attribute: 'width', type: Number })
+  canvasWidth = 1280;
+
+  @property({ attribute: 'height', type: Number })
+  canvasHeight = 720;
+
+  @property({ attribute: 'min-zoom', type: Number })
+  minimumZoom = 1;
+
+  @property({ attribute: 'max-zoom', type: Number })
+  maximumZoom = 1000;
+
+  @property({ attribute: 'background', type: String })
+  canvasBgColor = 'transparent';
+
+  @property({ attribute: 'link-color', type: String })
+  linkColor = '#fff';
+
+  @property({ attribute: 'border-accent-color', type: String })
+  borderAccent = '#F1C21B';
+
+  @property({ attribute: 'link-width', type: Number })
```

Comment:
Can we add descriptions to each property, good practice for others that may touch/view the code

```
  /**
   * Model ID or Model Name for which feedback is recording
   */
  @property({ attribute: 'model', type: String })
  private _model_id = '';
```

Reply:
Done.